Shaders/lilium__sdr_trc_fix.fx

```diff
@@ -69,38 +69,22 @@ uniform float BT1886_TARGET_BLACKPOINT
 
 float3 Bt1886(
   const float3 V,
-  const float  TargetInverseGamma)
+  const float gamma)
 {
-  static const float powLw = pow(BT1886_TARGET_WHITEPOINT, TargetInverseGamma);
-  static const float powLb = pow(BT1886_TARGET_BLACKPOINT, TargetInverseGamma);
+  static const float targetInverseGamma = 1.f / gamma;
+  static const float powLw = pow(BT1886_TARGET_WHITEPOINT, targetInverseGamma);
+  static const float powLb = pow(BT1886_TARGET_BLACKPOINT, targetInverseGamma);
 
   static const float powLw_minus_powLb = powLw - powLb;
 
-  static const float a = pow(powLw_minus_powLb, TARGET_POWER_GAMMA);
+  static const float a = pow(powLw_minus_powLb, gamma);
```

Comment:
you don't need to feed uniform variables to functions in ReShade fx code. they are always accessible. I know that it is not like that hlsl.